.gitlab-ci.yml

```diff
@@ -89,16 +89,20 @@ default:
     - /^pre-v[0-9]+\.[0-9]+-[0-9a-f]+$/
     - web
 
-.chaos-only:                       &chaos-only
+.build-chaos-only:                       &build-chaos-only
```

Comment:
OK, so now this will work as follows:
`(refs(master OR version OR pr OR web-triggered) AND variables(commit message contains))`
Does it correspond to how you want to run it?

Reply:
And `chaos-test-singlenodeheight:` `docker-build-chaos:` jobs will run on `(any time (not even PR) AND (variables(commit message contains))`. This is not right. Why make these conditions different?

Reply:
Yes, thanks, I double checked with gitlab documentation. This is getting complicated with the limitation of the ```only```. 

For the chaos deployment to proceed (now its able to spawn a various sized private network, it needs an image to be built and pushed to chaos docker repo)

I updated the ```build-only ``` to ```rules```, could you please verify it again?  thanks! i would love to change all the ```only``` if this is valid.